cli/cmd/install.go

```diff
@@ -142,7 +142,7 @@ func getRepo(name string, url string, settings *cli.EnvSettings) error {
 	repoFile := settings.RepositoryConfig
 
 	// Ensure the file directory exists as it is required for file locking
-	err := os.MkdirAll(filepath.Dir(repoFile), os.ModePerm)
+	err := os.MkdirAll(filepath.Dir(repoFile), 0755)
```

Comment:
Why 0755 here, but 0644 down below in line 194?

Reply:
Because this is a directory and you can't do much with a directory without X.